packages/cli/README.md

```diff
@@ -0,0 +1,150 @@
+| Option        | Description                                        |
+| ------------- | -------------------------------------------------- |
+| --foundryUrl  | URL for the Foundry stack [string]                 |
+| --application | Application resource identifier (rid) [string]     |
+| --token       | Foundry API token [string]                         |
+| --tokenFile   | Path to file containing Foundry API token [string] |
+
+`--token` and `--tokenFile` are mutually exclusive and only one can be passed. If neither are passed the CLI runtime looks for these environment variables in order `[FOUNDRY_TOKEN, FOUNDRY_SDK_AUTH_TOKEN]`. If a token is not specified using any of those options, the CLI will exit with a failure.
+
+---
+
+The `site` subcommand is not runnable without passing a specific subcommand i.e (`deploy` or `version`)
+
+### `deploy`subcommand
+
+The deploy subcommand allows users to upload new versions of their site and set it as deployed
+
+```
+npx @osdk/cli site deploy
+```
+
+Deploy options
+
+| Option       | Description                                                                    |
+| ------------ | ------------------------------------------------------------------------------ |
+| --directory  | Directory containing site files [string]                                       |
+| --version    | New version of site to deploy [string]                                         |
+| --uploadOnly | Upload new site version only without setting as live [boolean][default: false] |
+
+Auto Version options
+
+| Option         | Description                                                                                                                                                         |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --autoVersion  | Enable auto versioning [string][choices: "git-describe"]                                                                                                            |
+| --gitTagPrefix | Prefix to match git tags on when 'git-describe' auto versioning is used. If not provided, all tags are matched and the prefix 'v ' is stripped if present. [string] |
+
+`--version` and `--autoVersion` are mutually exclusive and only one can be passed.
+
+If `git-describe` is used for `--autoVersion`, the CLI will try to infer the version by running the `git describe` command with optionally `--match=<gitTagPrefix>` set if `--gitTagPrefix` is passed.
+
+#### `foundry.config.json` config file
+
+This file contains project related configuration for Foundry website hosting. If a valid project file is present in the directory where the CLI is run, the CLI will automatically load the project file and depending on the present data, the CLI will be able to apply default values for otherwise required options.
+
+An example of a full `foundry.config.json` is show below:
```

Comment:
Some wording changes and thoughts on moving this to a top level section as it doesn't just apply to the deploy subcommand? We could put it directly after "Running locally" for most visibility or after the subcommand documentation for less visibility

```suggestion
## Project config file

A `foundry.config.json` file may be used in your project's root to simplify usage of the CLI by populating default values for commands. The current working directory will be traversed up to the root directory and the first `foundry.config.json` file if found will be used.

This file contains project related configuration for Foundry website hosting. If a valid project file is present in the directory where the CLI is run, the CLI will automatically load the project file and depending on the present data, the CLI will be able to apply default values for otherwise required options.

An example of a full `foundry.config.json` is shown below:
```